### PR TITLE
Collect data from more stages

### PR DESCRIPTION
**Pull request by blindpirate**
So that we can analyze the feedback time of more stages.

### CODE REVIEW

File: teamcity-export/src/main/kotlin/org/gradle/devprod/collector/teamcity/TeamcityClientService.kt [hunk 1]
```diff
@@ -12,21 +12,30 @@ import java.time.temporal.ChronoUnit
 class TeamcityClientService {
     private
     val teamCityInstance: TeamCityInstance = TeamCityInstanceFactory.guestAuth("https://builds.gradle.org")
+
     private
     val pipelines = listOf("Master", "Release")
+
     private
-    fun readyForNightlyFor(pipeline: String) = BuildConfigurationId("Gradle_${pipeline}_Check_Stage_ReadyforNightly_Trigger")
+    val buildConfigurations = listOf(
```
Comment (wolfs) [inline teamcity-export/src/main/kotlin/org/gradle/devprod/collector/teamcity/TeamcityClientService.kt:20]: Why not use something like `fun buildConfigurationsFor(pipeline: String)`, then you don't need to do the string manipulation later on?

**Review by wolfs — COMMENTED** [review]
Looks good, I left one comment.

**Comment by blindpirate** [comment]
@wolfs PTAL!

**Review by wolfs — APPROVED** [review]
LGTM!

**Pull request merged**